Dockerfile

```diff
@@ -87,7 +90,5 @@ ENV NODE_EXTRA_CA_CERTS='/azp/certchain/jst_combined.pem'
 ENV HOME='/home'
 ENV JAVA_HOME_11_X64='/usr/lib/jvm/java-11-openjdk-amd64'
 
-### user name recognition at runtime w/ an arbitrary uid - for OpenShift deployments
-ENTRYPOINT [ "uid_entrypoint" ]
```

Comment:
Leave this in but commented out? Since this is essential to get this working on an openshift environment (and the repo is for openshift-azure-devops-agents after all ;) )

Reply:
Sourcing uid_entrypoint is moved to start.sh (see lines 95-99) to comply with https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/docker?view=azure-devops using entrypoint for "--once"